platform/iphone/gl_view.mm

```diff
@@ -606,6 +634,18 @@ - (id)initWithFrame:(CGRect)frame {
 				   name:AVAudioSessionRouteChangeNotification
 				 object:nil];
 
+	printf("******** adding observer for keyboard show/hide\n");
```

Comment:
Should this maybe use the new logging infrastructure?

Reply:
This is invoked at the very beginning - even before OS is initialized, so there is no infrastructure at that point.

Reply:
yeah, it looks like there are more printf()'s like this in iphone os, sorry.